tests/www/approvals_views/test_prolongation.py

```diff
@@ -133,6 +135,37 @@ def test_prolong_approval_view_bad_reason(self):
         )
         self.assertNotContains(response, self.MAX_DURATION_TEXT)
 
+    @freeze_time("2023-08-18")
```

Comment:
Le `freeze_time` est utile ? En première lecture je m'était dit que c'était pour toujours avoir les mêmes dates mais en relisant on s'en fiche, non ?

Reply:
C’est pour éviter que le test ne casse vers minuit.

Reply:
:+1:, alors peut-être utiliser la forme qui ne spécifie pas de valeur afin d'éviter d'avoir une valeur magique ?
```suggestion
    @freeze_time()
```